src/components/pages/form-details-preview-page/FormDetailsPreviewPage.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { FormDetails } from 'components/_common/form-details/FormDetails';
+import { ContentCommonProps } from 'types/content-props/_content-common';
+import { FormDetailsPageProps } from 'types/content-props/form-details';
+
+import styles from './FormDetailsPreviewPage.module.scss';
+
+export const FormDetailsPreviewPage = ({
+    data,
+}: FormDetailsPageProps & ContentCommonProps) => (
```

Comment:
Kan `ContentCommonProps` inkluderes i `FormDetailsPageProps`-typen, slik at den følger samme pattern som for de andre content typene?